### ISSUE TITLE
 Cant Host on Android

### ISSUE THREAD

**Issue by Rreojs** [jsettlers/settlers-remake#794]
I press start, pick a map, but then my game just crashes.

**Comment by paulwedeck** [comment]
Yes, the default server is currently not running and there is not (yet (give me a few weeks)) an option for android to select a different server.
You can currently only host and join games via the swing/desktop interface ([setup](https://github.com/jsettlers/settlers-remake/issues/793)).

**Comment by Rreojs** [comment]
> Yes, the default server is currently not running and there is not (yet (give me a few weeks)) an option for android to select a different server.
> You can currently only host and join games via the swing/desktop interface ([setup](https://github.com/jsettlers/settlers-remake/issues/793)).
> 
> 

Im happy that you still work on this project. Keep it up, you got a new fan!
